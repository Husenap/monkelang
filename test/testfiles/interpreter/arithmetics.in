monke say 1+2
monke say 1-2+3
monke say 1.3-2.2+3.5
monke say 2+3*4
monke say 4/2
monke say 4%3
monke say 2==2
monke say 2==1
monke say 2!=2
monke say 2!=1
monke say 1<2
monke say 2<2
monke say 1<=2
monke say 2<=2
monke say 3<=2
monke say 2>2
monke say 3>2
monke say 1>=2
monke say 2>=2
monke say 3>=2
monke say "Hello, " + "World"
monke say "I am " + 26 + " years old."

