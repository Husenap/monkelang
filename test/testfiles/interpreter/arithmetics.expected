3
2
2.6
14
2
1
true
false
false
true
true
false
true
true
false
false
true
false
true
true
Hello, World
I am 26 years old.
